troubleshoot/observability/inspect.md

```diff
@@ -1,6 +1,10 @@
 ---
 mapped_pages:
   - https://www.elastic.co/guide/en/observability/current/inspect.html
+applies_to:
+  stack: all
+  serverless:
+    observability: all
```

Comment:
I don't think _Inspect_ is available in serverless. @elastic/obs-docs can anyone confirm?

```suggestion
```

Reply:
Confirmed

Reply:
Wait. It is!

<img width="820" alt="Screenshot 2025-03-12 at 2 38 24 PM" src="https://github.com/user-attachments/assets/704c8f04-7104-4cd2-86f4-c02311e8968f" />

Reply:
I had to enable it to see it (like the docs state 🙃)